src/components/VSnackbar/VSnackbar.ts

```diff
@@ -55,17 +60,17 @@ export default {
 
   methods: {
     setTimeout () {
-      clearTimeout(this.activeTimeout)
+      window.clearTimeout(this.activeTimeout)
```

Comment:
window shouldn't be needed for these afaik

Reply:
seemed like without it typescript was wanting to use the node versions, which returned `NodeJS.Timer`

Reply:
yeah weird - I get `number` from `setTimeout(() => 0)`, but `NodeJS.Timer` if I add the second argument.